### PR TITLE
T1550

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @benrito received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/118fde5890affe4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/118fde5890affe4/output.txt

Total script time: 0.54 mins

Server is running on http://pr539.butyr.org/templates/test.html

File: templates/supported/simple/butter/template.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
+
+/* This Source Code Form is subject to the terms of the MIT license
+ * If a copy of the MIT license was not distributed with this file, you can
+ * obtain one at http://www.mozillapopcorn.org/butter-license.txt */ (function (Butter) {
+    Butter({
```
Comment (humphd) [inline templates/supported/simple/butter/template.js]: This needs to get wrapped in document.addEventListener( "DOMContentLoaded", function() { ... }, false); or Butter will blow up.

**Pull request merged**